test/lib/mayaUsd/render/mayaToHydra/testUVandUDIM.py

```diff
@@ -39,5 +39,11 @@ def test_UDIMs(self):
         self.setHdStormRenderer()
         self.verifySnapshot("quads_with_UDIMs.png")
 
+    def test_HDR(self):
+        mayaUtils.openTestScene("testUVandUDIM", "testHDR.ma")
+        self.setHdStormRenderer()
+        cmds.refresh()
+        self.assertSnapshotClose("quads_with_HDR.png", 0.04, 0.2)
```

Comment:
For consistency, I would suggest we either use and adjust the existing threshold constants, or if we use bespoke thresholds per-test as here, then we should have the other test follow the same structure as well and adjust its code to mirror this one